scripts/qa_crowbarsetup.sh

```diff
@@ -1204,15 +1210,16 @@ function onadmin_set_source_variables
             CLOUDSLE12DISTISO="SUSE-OPENSTACK-CLOUD-6-$arch*1.iso"
             CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-6-official"
         ;;
-        GMC*|M?)
+        GM7|GM7+up|GMC*|M?)
```

Comment:
In this case I would like to keep it here as the code right below this line shows what to do when cloud8 is available.